### ISSUE TITLE
Failing train - pedestrian detect model #object-detection

### ISSUE THREAD

**Issue by skyzhao3q** [NVIDIA/DIGITS#1052]
I created a dataset with setting `custom class` of  **'dontcare,pedestrian'** and then follow the steps of the tutorial for training object-detect model.But It is Failed.
Here is the result.
![image](https://cloud.githubusercontent.com/assets/3965470/18371996/e88e5854-7673-11e6-9256-ddba0a0b861b.png)

However I successfully created the model of dataset with `custom class` of **'dontcare,car'**.Nothing others is different from the **'pedestrian detect'** aboved.
![image](https://cloud.githubusercontent.com/assets/3965470/18372131/22070ea4-7675-11e6-88a8-a30a1e4f968f.png)

What am I missed ?  Anyone who tried this successfully ?

**Comment by skyzhao3q** [comment]
The first thing I thought is that the amount of the training data is different between 'car' and 'pedestrian'.
So I counted the label records, and got a result as below.
**car: 32737; pedestrian: 2615**
Is this the reason ?

**Comment by sherifshehata** [comment]
did you try it for more epochs? maybe it just needs time

**Comment by sherifshehata** [comment]
also try following the suggestion in this comment: https://github.com/NVIDIA/DIGITS/issues/980#issuecomment-242741587

it helps much

**Comment by skyzhao3q** [comment]
Anyway I have make the pedestrian model successfully.
Thinking someone may want to see how to do that.I created a repository below with the dataset , custom network and settings.  
https://github.com/skyzhao3q/NvidiaDigitsObjDetect
